sdks/python/setup.py

```diff
@@ -105,7 +105,7 @@ def get_version():
     'oauth2client>=2.0.1,<5',
     # grpcio 1.8.1 and above requires protobuf 3.5.0.post1.
     'protobuf>=3.5.0.post1,<4',
-    'pytz>=2018.3',
+    'pytz==2018.4',
```

Comment:
How about:
`pytz>=2018.3,<=2018.4`

There is no reason for us to not support both versions. It gives more flexibility to end users.

Reply:
Do you think `==2018.4` will break existing users when they upgrade to Beam 2.6.0?
How do we determine the lower bound for pytz versions?

Reply:
I do not think it will break existing users. The scenario I have in mind is, suppose there is a library X that pins pytz==2018.3 and we pin it at 2018.4. With that a user cannot have both X and Beam in the same environment. Since we already know that Beam works with both 2018.3 and 2018.4; it would be more flexible to work with either version of the dependency.

Reply:
Okay, I don't mind either way.